### ISSUE TITLE
Extending cloze card creation beyond one sentence

### ISSUE THREAD

**Issue by golthmike** [fonol/anki-search-inside-add-card#142]
Hi fonol! Thank you for this great tool :)
I was wondering if you could add the option to include more than one sentence in the cloze card creation generator. Many of the concepts I study require me to put a few sentences into one cloze card.
Thank you for considering :D

**Comment by fonol** [comment]
Hi, do you mean like when you select, it will not only include the sentence around your selection, but the sentence before and the sentence after?

**Comment by golthmike** [comment]
Yes, sth like that :) Now you select a word/several words and your add-on will automatically pull the sentence around it. One idea would be to reverse that. Select the text you want on the card and then create blanks within that text. Like that you have the choice how much you want to put onto the card. I guess its more a feature request :)

**Comment by fonol** [comment]
Yea it would be cool, but also some work. I'll put it on my list of future nice-to-have features :)

**Comment by golthmike** [comment]
Awesome. Thank you!

**Comment by p4nix** [comment]
> Yes, sth like that :) Now you select a word/several words and your add-on will automatically pull the sentence around it. One idea would be to reverse that. Select the text you want on the card and then create blanks within that text. Like that you have the choice how much you want to put onto the card. I guess its more a feature request :)

What is the disadvantage of marking the text to copy and paste it to the according cloze field? I would tend to say that it is about as quick as your proposed solution.

**Comment by golthmike** [comment]
> > Yes, sth like that :) Now you select a word/several words and your add-on will automatically pull the sentence around it. One idea would be to reverse that. Select the text you want on the card and then create blanks within that text. Like that you have the choice how much you want to put onto the card. I guess its more a feature request :)
> 
> What is the disadvantage of marking the text to copy and paste it to the according cloze field? I would tend to say that it is about as quick as your proposed solution.

**Comment by golthmike** [comment]
> > Yes, sth like that :) Now you select a word/several words and your add-on will automatically pull the sentence around it. One idea would be to reverse that. Select the text you want on the card and then create blanks within that text. Like that you have the choice how much you want to put onto the card. I guess its more a feature request :)
> 
> What is the disadvantage of marking the text to copy and paste it to the according cloze field? I would tend to say that it is about as quick as your proposed solution.

Copy and pasting is much slower in my opinion :)
